### PR TITLE
Added filtering by email for the commits REST-endpoint and a front-end date-range filter.

### PR DESCRIPTION
**Pull request by maximAtanasov**
Closes issue #115

### CODE REVIEW

File: coderadar-test/src/main/asciidoc/restapi.adoc [hunk 1]
```diff
@@ -562,6 +562,8 @@ include::{snippets}/commit/list/response-fields.adoc[]
 
 include::{snippets}/commit/list/http-request.adoc[]
 
+NOTE: An additional "email" parameter may be supplied to this request to only return the commits authored by the matching contributor.
```
Comment (KilianKrause) [inline coderadar-test/src/main/asciidoc/restapi.adoc:565]: Maybe include an `example request` and `example response` here.

File: coderadar-test/src/test/java/io/reflectoring/coderadar/rest/query/GetCommitsInProjectControllerTest.java [hunk 2]
```diff
@@ -74,6 +74,27 @@ void returnsAllCommitsInProject() throws Exception {
     Assertions.assertEquals("modify testModule1/NewRandomFile.java", commits.get(0).getComment());
   }
 
+  @Test
+  void returnsAllCommitsInProjectForContributor() throws Exception {
```
Comment (KilianKrause) [inline coderadar-test/src/test/java/io/reflectoring/coderadar/rest/query/GetCommitsInProjectControllerTest.java:78]: Document this test (request + response) and include it in the `restapi.adoc`-file.

File: coderadar-ui/package.json [hunk 3]
```diff
@@ -3,7 +3,7 @@
   "version": "0.5.0",
   "scripts": {
     "ng": "node --max_old_space_size=5048 ./node_modules/@angular/cli/bin/ng",
-    "start": "node --max_old_space_size=5048 ./node_modules/@angular/cli/bin/ng serve --prod",
+    "start": "node --max_old_space_size=5048 ./node_modules/@angular/cli/bin/ng serve",
```
Comment (KilianKrause) [inline coderadar-ui/package.json:6]: Please keep the `--prod` flag

File: coderadar-test/src/test/java/io/reflectoring/coderadar/rest/query/GetCommitsInProjectControllerTest.java (hunk 2 above)
Comment (maximAtanasov) [reply coderadar-test/src/test/java/io/reflectoring/coderadar/rest/query/GetCommitsInProjectControllerTest.java:78]: Done

**Pull request merged**